pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -171,3 +175,21 @@ func (m *defaultVirtualCenterManager) IsExtendVolumeSupported(ctx context.Contex
 	}
 	return !is67u3Release, nil
 }
+
+// IsOnlineExtendVolumeSupported checks if online extend volume is supported or not.
+func (m *defaultVirtualCenterManager) IsOnlineExtendVolumeSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+
+	// Get VC instance
+	vcenter, err := m.GetVirtualCenter(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to get vCenter. Err: %v", err)
+		return false, err
+	}
+	vCenterVersion := vcenter.Client.Version
+	if vCenterVersion != cns.ReleaseVSAN67u3 && vCenterVersion != cns.ReleaseVSAN70 && vCenterVersion != cns.ReleaseVSAN70u1 {
+		return true, nil
```

Comment:
Since you are not handling ESX version check in this PR, could you please create an internal bug to track that work? I think we should fix this in the next immediate patch release as its very common that the customers upgrades the vCenter first and then the ESX.

Reply:
Created an internal bug and assigned it to myself.